src/clog/core.clj

```diff
@@ -11,7 +11,7 @@
    :headers {}
    :body page})
 
-(defn- save-request [parameters]
+(defn- save-blog-request [parameters]
```

Comment:
you changed the method name but the test still refers to save-blog?  Did the tests pass on this commit?